dags/sources/tasks/transform/transform_df.py

```diff
@@ -218,6 +220,18 @@ def get_latlng_from_geopoint(row: pd.Series, _) -> pd.Series:
     return row[["latitude", "longitude"]]
 
 
+def compute_location(row: pd.Series, _):
```

Comment:
```suggestion
def compute_location(row: pd.Series):
```

Voir s'il y a moyen de définir dag_config en kwargs lors de l'appel à la fonction `compute_location`